contents/tutorials/how-to-run-successful-group-experiments.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to run successful group-level A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: // what to put here?
```

Comment:
I usually put the current date to start (so it shows up at the top locally) then change before publishing.